sfcasts/ep1/computed.md

```diff
@@ -1,228 +1,192 @@
-# Computed
+# Computed Properties
 
 Our collapsing sidebar works, but *yikes*, it looks terrible! We *need* to fix this!
-What we need to do here is hide all of these texts and links entirely when the sidebar
-is in its collapsed state. 
+Let's hide the text and links entirely when the sidebar
+is in its collapsed state.
 
 ## v-if directive
 
-Look at `sidebar.vue` in the template, we want to hide the `<h5>`, the `<ul>` and the
-`<hr />`, basically everything except for the button, when we're in a collapsed
-state.
+Look at `sidebar.vue`. In the template, we want to hide the `<h5>`, the `<ul>` and the
+`<hr />`: basically everything except for the button.
 
 In `Vue`, there are two ways to do this and both of them come in the form of
-*directives*. At the end of this chapter we will have seen almost all of the important
-directives in `Vue`. You will be a Directive Master!
```

Comment:
I could have gone either way on this :). On the one hand, we're telling them "Hey! There aren't 1000 directives, you've seen almost all of the important ones". On the other hand, we're adding noise that's delaying us talking about our real object. That was the conflict in my head here.

Reply:
Agree 100%